src/libraries/System.Collections.Immutable/src/System/Collections/Frozen/FrozenSet.cs

```diff
@@ -21,6 +24,11 @@ namespace System.Collections.Frozen
     /// </remarks>
     public static class FrozenSet
     {
+        // TODO: these need to be tuned through benchmarks to find the right values
+        private const int MaxItemsInSmallSet = 3;
+        private const int MaxItemsInSmallIntegerSet = 8;
+        private const int MaxSparsenessFactorInSparseRangeIntegerSet = 8;
```

Comment:
I don't imagine we'd want these values to diverge from their equivalents in the dictionary implementation.  Can we dedup them and put the shared constants in some more centralized location?

Reply:
Done.